firebolt_provider/hooks/firebolt.py

```diff
@@ -80,26 +61,34 @@ class FireboltHook(DbApiHook):
     conn_type = "firebolt"
     hook_name = "Firebolt"
 
+    @staticmethod
+    def get_connection_form_widgets() -> Dict[str, Any]:
+        from flask_appbuilder.fieldwidgets import BS3TextAreaFieldWidget, BS3TextFieldWidget
+        from flask_babel import lazy_gettext
+        from wtforms import BooleanField, StringField
+
+        return {
+            "account_name": StringField(lazy_gettext("Account"), widget=BS3TextFieldWidget()),
+        }
```

Comment:
What does this do, add a field in the UI?

Reply:
Yes, this creates an additional field at the end of the view with provided widget